src/js/utils/StringUtil.js

```diff
@@ -1,6 +1,14 @@
 import marked from 'marked';
 
 const StringUtil = {
+  arrayToCommaSeparatedString: function (array) {
```

Comment:
Good call. I only extracted that one in to a function as I was repeating myself over and over again. I'll generalize the method.